content/en/docs/setup/independent/high-availability.md

```diff
@@ -40,14 +48,22 @@ For both methods you need this infrastructure:
   requirements](/docs/setup/independent/install-kubeadm/#before-you-begin) for
   the workers
 - Full network connectivity between all machines in the cluster (public or
-  private network is fine)
-- SSH access from one device to all nodes in the system
+  private network)
 - sudo privileges on all machines
+- SSH access from one device to all nodes in the system
+- `kubeadm` and `kubelet` installed on all machines. `kubectl` is optional.
 
 For the external etcd cluster only, you also need:
 
 - Three additional machines for etcd members
 
+Optional for all control plane nodes:
+
+- Install [kubeadm-getter](https://github.com/neolit123/kubeadm-getter/releases).
```

Comment:
@neolit123 @timothysc 
are thee plans to get this tool under the sig umbrella? what kind of guarantee are we providing to the users?

Reply:
it's a left over and should be removed from the PR.
we can discuss it later on a sig-meeting.